src/public/index.html

```diff
@@ -0,0 +1,27 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Notifications</title>
+    <style>
+     a {
+        text-decoration:none;
+     }
+    </style>
+</head>
+<body> 
+    <script src="/socket.io/socket.io.js"></script>
+    <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
+    <script>
+        const socket = io.connect('http://localhost:3000');
+        socket.emit('connect_user', '0e22ed1c-a1a5-4f49-a4ca-000732bfa49o');
+        socket.on('new-trip-request', (notification) =>  {
+            $("#notifications").append(`<li><a href="#">${notification.body}</a></li>`);
+        });
+    </script>
+    <h1>Notifications</h1>
+    <ul id="notifications">
+
+    </ul>
+</body>
+</html>
```

Comment:
```suggestion
</html>

```
Consider add a new line to the end